src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -230,48 +230,48 @@ LedgerConsensusImp::LedgerConsensusImp (
         LocalTxs& localtx,
         LedgerMaster& ledgerMaster,
         FeeVote& feeVote)
-    : app_ (app)
```

Comment:
The only thing I'll add here is that this convention must be careful not to use a capital letter as the first letter after the underscore.  Such an identifier is reserved by the implementation (for example `_App` could be a macro defined by the compiler or std::lib).  `_lower_case` names are also reserved by the implementation, but only in the global scope, which won't conflict with this convention (since we aren't in the global scope).

My words are not intended to be an argument for or against this convention, just to communicate relevant information in this territory.